src/odemis/gui/cont/stream_bar.py

```diff
@@ -1198,6 +1200,51 @@ class SecomStreamsController(StreamBarController):
     Controls the streams for the SECOM and DELPHI live view
     """
 
+    def __init__(self, *args, **kwargs):
+        dir = os.path.join(get_home_folder(), u".config/odemis")
+        file_path = os.path.abspath(os.path.join(dir, f"stream_settings.json"))
+        max_entries = 10  # Maximum number most recent settings to display
+        # Initialize a file to save acquired streams
+        self.stream_settings = StreamSettings(file_path, max_entries)
+        super().__init__(*args, **kwargs)
```

Comment:
Typically, it is suggested to call the super function as first thing in the function, unless there is a specific reason not to. I think here you can call it first.